package.json

```diff
@@ -19,8 +19,10 @@
     "THIRD-PARTY-NOTICES"
   ],
   "scripts": {
-    "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run api-extractor && npm run generate-docs && npm run generate-notices && vite build",
-    "lint": "eslint .",
+    "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run format-check && npm run api-extractor && npm run generate-docs && npm run generate-notices && vite build",
+    "prepare": "husky install",
+    "format-check": "prettier --check . && eslint .",
+    "format-fix": "prettier --write . && eslint . --fix",
```

Comment:
Thoughts on changing `format-fix` to just `clean`?

Reply:
I found format-fix is a better name than clean, since we also have format-check. Do you have a reason for changing? `clean` does not look obvious to me on what it does at the first glance.

Reply:
Nah, just thinking about it. What is there is fine.